.buildkite/pipeline.yml

```diff
@@ -23,7 +23,7 @@ steps:
         nvm install && nvm use
         npm ci --unsafe-perm --prefer-offline --no-audit --no-progress
 
-        npm run prebundle:js
+        # npm run prebundle:js
```

Comment:
The command that updates the i18n localizations uses WP-CLI that requires PHP, so we would need to update the [`gb-mobile-image` docker image](https://github.com/wordpress-mobile/docker-gb-mobile-image) and include PHP as part of the installed tools.

Reply:
This will be solved in https://github.com/wordpress-mobile/docker-gb-mobile-image/pull/3.